releases/point-2022-2-3-release-note.md

```diff
@@ -0,0 +1,38 @@
+---
+description: This article lists improvements and bug fixes in SysKit Point version 2022.2.3
+--- 
+
+# SysKit Point 2022.2.3
+
+[Schedule a demo](https://www.syskit.com/products/point/request-a-demo/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2022.2.3
+
+**Build number:** 2022.2.3.8
+
+**Release date:** Mar 31, 2022
+
+## Get the Latest Version
+
+* **New to SysKit Point?** Get the latest SysKit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point).<br/>
+    Find all steps described in detail in the [deployment article](../installation/deploy-syskit-point.md).
+    
+* **Already using SysKit Point?** Upgrade to the latest SysKit Point version. <br/>
+[![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FUpdateFilesARM%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../installation/upgrade-syskit-point.md).
+
+## Improvements & Bug Fixes
+
```

Comment:
možemo dodati i ovaj link za članak što je Igor stavio u komentar. pa da se može odmah posjetiti članak za RBS.

Learn more about Role-based Access in this article. -> link je https://www.syskit.com/products/point/documentation/?doc_page=configuration/enable-role-based-access.md#syskit-point-collaborators

u ovome linkanom članku treba isto staviti ovu novu opciju + sliku novu .. al to je prijavljeno mislim već